### ISSUE TITLE
About Test Result

### ISSUE THREAD

**Issue by infinitycan** [sthoduka/qvpipe_classification#1]
Thanks for your code.
I could get the best val_mAP = 53.31 at logs/lightning_logs/version_9/checkpoints/epoch=40-step=410.ckpt , and i also use it as the checkpoint to test with default batch-size = 64 but only got the mAP= 5.8 .i don't know that's why
![image](https://github.com/user-attachments/assets/4a90337b-7a55-4a4d-8aa6-61040f7b2c02)

**Comment by sthoduka** [comment]
hi @HJiaC, an mAP of 53.31 sounds reasonable. For the challenge, the final result I got was based on combining the results from three runs I think (each with mAP 58.0, 56.47 and 54.04). So maybe with some additional fine-tuning, you would reach those values.

I didn't quite understand how you got the mAP of 5.8. What was the difference compared to the run where you got 53.31?

**Comment by sthoduka** [comment]
Btw, for the test set, the labels are set to 0, since we don't have the ground truth labels (see [here](https://github.com/sthoduka/qvpipe_classification/blob/master/datasets/qvpipe_dataset.py#L144)).

**Comment by HJiaC** [comment]
Thanks for your reply! 
I am a postgraduate in China Mainland and my research interests are in pipeline anomaly detection and localization.
What you mentioned in letter is why I got the mAP of 5.8.I just execute the test.sh without noticing that the labels of test videos are set to default(0).Since i can’t get the ground truth of test videos,  I just separate the qv_train.json(6399 videos and corresponding labels) into train set(4000)、val set(399)、test set(2000).Finally,I got the test_mAP of 39.47.I think this result is normal, but need more train epochs to get a better result.

**Comment by infinitycan** [comment]
> Btw, for the test set, the labels are set to 0, since we don't have the ground truth labels (see [here](https://github.com/sthoduka/qvpipe_classification/blob/master/datasets/qvpipe_dataset.py#L144)).

Thanks for your reply! 
I am a postgraduate in China Mainland and my research interests are in pipeline anomaly detection and localization.
What you mentioned is why I got the mAP of 5.8.I just execute the test.sh without noticing that the labels of test videos are set to default(0).Since i can’t get the ground truth of test videos,  I just separate the qv_train.json(6399 videos and corresponding labels) into train set(4000)、val set(399)、test set(2000).Finally,I got the test_mAP of 39.47.I think this result is normal, but need more train epochs to get a better result.
